src/Infrastructure.EntityFramework/EntityFrameworkCache.cs

```diff
@@ -0,0 +1,315 @@
+﻿using Bit.Infrastructure.EntityFramework.Models;
+using Bit.Infrastructure.EntityFramework.Repositories;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Bit.Infrastructure.EntityFramework;
+
+public class EntityFrameworkCache : IDistributedCache
+{
+#if DEBUG
+    // Used for debugging in tests
+    public Task scanTask;
+#endif
```

Comment:
Made it so the scanTask is available in tests so we can actually make sure that it doesn't end in a fault state. Since the Task is largely fired off and not handled it could error and we never get it reported. This could still happen in production code though.